### DISCUSSION TITLE
wpf有没有好用的属性编辑器，可直接编辑实例的属性，不用写ui

### DISCUSSION THREAD

**Discussion by designerlast** [BYJRK/DotNet-Discussions#287] [WPF 开发]
wpf有没有好用的属性编辑器，可直接编辑实例的属性，不用写ui的

**Comment by BYJRK** [comment]
你指的是属性面板（PropertyGrid）吗？比如 WinForm 自带的。WPF 有一些控件库提供了这个，比如：

- [HandyControl](https://github.com/HandyOrg/HandyControl)
- [Extended.Wpf.Toolkit](https://github.com/xceedsoftware/wpftoolkit)
